File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    half_result = ''
    for i in range(1, len(food)):
        count = food[i] // 2
        for j in range(count):
            half_result += str(i)
        
    return half_result + '0' + half_result[::-1]
        
        
        

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
const solution = (food) => {
    let half_result = '';
    food.forEach((i, idx) => {
        if (idx !== 0) {
            half_result += String(idx).repeat(Math.floor(i / 2));
        }
    });
    return half_result + '0' + half_result.split('').reverse().join('');
}